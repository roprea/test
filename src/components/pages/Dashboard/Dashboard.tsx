import { memo } from 'react';
import type { HTMLAttributes } from 'react';

function Dashboard(props: Props) {
  return <div {...props}>Dashboard Content</div>;
}

type Props = HTMLAttributes<HTMLElement>;

export default memo(Dashboard);
