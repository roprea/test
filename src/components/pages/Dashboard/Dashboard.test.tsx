import { basicTestRunner } from 'test-utils/basic-test-runner';
import Dashboard from './Dashboard';

describe('Dashboard', () => {
  basicTestRunner(<Dashboard />);
});
