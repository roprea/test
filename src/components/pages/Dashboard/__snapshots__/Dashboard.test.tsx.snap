// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Basic component behaviour should match snapshot 1`] = `
<div>
  <div>
    Dashboard Content
  </div>
</div>
`;
